src/main/scala/models/SupportedType.scala

```diff
@@ -3,35 +3,26 @@ package models
 import java.io.{File, Reader}
 import scala.io.Source
 
+/*
+Supported Types. Each are handled differently.
+
+ParsableType are ready to be parsed
+NonParsableType need tuning before being ready to be parsed
```

Comment:
I like the idea a lot but I just don't like these names

Reply:
TraversableType, NonTraversableType?

Reply:
What Pedro said